### PR TITLE
HTML/CSS & JavaScript capstone project

### PR DESCRIPTION
**Pull request by waelelsafty07**
- The home page should have a link in the menu to the about page.
- The logo in the header links to the home page.
- When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.
- The section "Featured speakers" should be created dynamically in JavaScript.
- You should use a JavaScript variable with the data about the speakers and use it when the page loads to create the HTML for this section dynamically.

**Project documentation**
video [loom](https://www.loom.com/share/012d3addb61c46b999f4db6419e86829)

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+    `,
+    image: 'images/Massimo Bottura.webp',
+  },
+  {
+    name: 'Dominique Crenn',
+    job: 'Chef/Owner, Atelier Crenn (San Francisco, USA)',
+    description: `
+    Dominique Crenn is a French-American chef and restaurateur, and the first female chef in the US to earn three Michelin stars. She is the owner of Atelier Crenn, a highly acclaimed restaurant in San Francisco, where she creates innovative, multi-course tasting menus that reflect her personal experiences and emotions. Crenn is known for her commitment to sustainability and her efforts to promote diversity and inclusion in the culinary industry.`,
+    image: 'images/Dominique Crenn.jpg',
+  },
```
Comment (aamirkhan2478) [inline js/index.js:24]: - [ ] Please fix the size of the `feature cooks` images if you check your design the size of `cooks` is very small and it does not match with Bahance design so kindly fix it :+1: check below your design and Behance design :slightly_smiling_face: 

**Your Design**
Screenshot from 2023-04-20 19-52-37

**Bahance Design**
Screenshot from 2023-04-20 19-52-23

- [ ] Please reduce the length of the paragraph then it looks good and professional :+1:

File: index.html [hunk 2]
```diff
@@ -4,9 +4,226 @@
...
+                    <div class="description mb-2">
+                        Experience a special tasting menu featuring signature dishes from each of our 4 top chefs. Each dish will be introduced by the chef who created it, giving you a chance to learn more about their culinary inspirations and techniques.
+                    </div>
+                </div>
+                <div class="col-md d-flex flex-sm-column flex-md-column align-items-center opacity">
+                    <div class="icon mb-2 me-2">
+                        <i class="fa-solid fa-people-group"></i>
+                    </div>
+                    <div class="title mb-2 me-2 orange-color">
+                        Meet the Chefs
```
Comment (aamirkhan2478) [inline index.html:113]: - [ ] Please fix the spacing issue in your `main program` section if you check your `main program mobile version` section there are lots of spacing issues so kindly fix this issue :+1: check below :point_down: the comparisons between your design and Bahnace design :slightly_smiling_face: 

**Your Design**
screencapture-127-0-0-1-5500-index-html-2023-04-20-19_47_35 (1)

**Bahance Design**
Screenshot from 2023-04-20 19-50-10

**Review by aamirkhan2478 — CHANGES_REQUESTED** [review]
Hi @waelelsafty07 , 



Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Good Effort
✔️ Your design is good.
✔️ All linter passed

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/index.js [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+const createtext = (textData, tagElement) => {
+  const tag = document.createElement(tagElement);
+  const text = document.createTextNode(textData);
+  tag.appendChild(text);
+  return tag;
+};
+
+const createFeatures = (feature) => {
+  const columnDiv = createDiv('col-md-6 d-flex mt-5 cook-person');
+
```
Comment (ABDULALI3468) [inline js/index.js:64]: **[OPTIONAL]** Currently, the image is overflowing over the text please consider resolving the issue to meet the design guidelines ☑️ 

image

File: index.html [hunk 4]
```diff
@@ -4,9 +4,226 @@
...
+                <div class="col-3 image">
+                    <img src="images/logo-chef.png" alt="Hire Chef" />
+                </div>
+                <div class="col-9  copywrite">
+                    <p>Hire Chef. Some Rights Reserved.</p>
+                    <p class="p-0 m-0 description">The design used in this project is based on the original work by [Cindy Shin](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) and has been modified to fit the requirements of the project.</p>
+                </div>
+            </div>
+        </div>
+    </footer>
```
Comment (ABDULALI3468) [inline index.html:224]: As asked by the previous code reviewer please consider aligning the content inside the footer in the center to meet the design guidelines currently, it's okay in the desktop version but does not fit well in the mobile version.

## YOUR DESIGN:--


image
image

**Review by ABDULALI3468 — CHANGES_REQUESTED** [review]
# Changes Required 👍 

Hi @waelelsafty07 , 



Great work on making the changes requested by a previous reviewer 👏 
You've done really well implementing some of the requested changes, but there are still some which aren't been addressed yet.


## Highlights 
☑️ Well implemented design.
✅All linters are checked.
✅Project is professionally documented.
✅The right GitHub flow method has been used.
✅Professional pull request title and description.


 ## Required Changes ♻️

> **_Check the comments below to have a look at the required changes._**

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.

> * This is My **GitHub**: GitHub followers
> * This is My **Linkedin**: [Linkedin Badge](https://www.linkedin.com/in/abdul-ali-5400bb216/)&nbsp;
> * This is My **Twitter**: [Twitter Badge](https://twitter.com/mrabdul_ali)&nbsp;

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AnselemOdims — APPROVED** [review]
### Hi  @waelelsafty07 🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**